source/modules/MEI.shared.xml

```diff
@@ -5769,6 +5769,32 @@
       <p>The model of this element is based on the <ref target="https://tei-c.org/release/doc/tei-p5-doc/en/html/ref-extent.html">extent</ref> element of the Text Encoding Initiative (TEI).</p>
     </remarks>
   </elementSpec>
+  <elementSpec ident="extData" module="MEI.shared">
+    <gloss versionDate="2023-05-23" xml:lang="en">extended data</gloss>
+    <desc xml:lang="en">Provides a container element for non-MEI data formats.</desc>
+    <classes>
+      <memberOf key="att.basic" />
+      <memberOf key="att.labelled" />
+      <memberOf key="att.responsibility" />
+      <memberOf key="att.typed" />
+      <memberOf key="att.whitespace"/>
+      <memberOf key="att.pointing"/>
+      <memberOf key="att.internetMedia"/>
+    </classes>
+    <content>
+      <rng:zeroOrMore>
+        <rng:group>
+          <rng:choice>
+            <rng:text/>
+            <rng:ref name="macro.anyXML"/>
+          </rng:choice>
+        </rng:group>
+      </rng:zeroOrMore>
+    </content>
+    <remarks xml:lang="en">
+      <p>Container for holding non-MEI data formats, similar to @extMeta but available in @when rather than in MEI.header. @extData allows for data from audio or other sources to be linked to notes or other score events. Data should be enclosed in a CDATA tag.</p>
```

Comment:
<p>Container for holding non-MEI data formats, similar to @extMeta but available in @when rather than in MEI.header. @extData allows for data from audio or other sources to be linked to notes or other score events. When the data contains left angle bracket (less-than) or ampersand characters, or when it contains significant white space that should be preserved (such as line breaks), then the data should be enclosed in a CDATA section (e.g., for JSON formatted data).</p>